eradiate/radprops/rad_profile.py

```diff
@@ -29,8 +30,17 @@
 from ..validators import all_positive
 
 
+def _to_quantity(variable):
+    try:
+        units = variable.units
```

Comment:
I didn't know one could reference metadata as regular attributes, that's nice!

Reply:
Indeed, I like that a lot :)